src/components/ComparisonPointFieldArray/ComparisonPointFieldArray.js

```diff
@@ -100,15 +100,21 @@ class ComparisonPointFieldArray extends React.Component {
             name={`${name}[${index}]`}
             onComparisonPointSelected={selectedComparisonPoint => this.handleComparisonPointSelected(index, selectedComparisonPoint, comparisonType)}
           />
-          <Field
-            component={Datepicker}
-            defaultValue={this.state.currentDate}
-            id={`data-test-field-date-${comparisonType}`}
-            index={index}
-            label={<FormattedMessage id="ui-erm-comparisons.newComparison.onDate" />}
-            name={`${name}[${index}].onDate`}
-            parser={this.parseDateOnlyString}
-          />
+          <Row>
+            <Col xs={3}>
+              <Field
+                component={Datepicker}
+                defaultValue={this.state.currentDate}
+                id={`data-test-field-date-${comparisonType}`}
+                index={index}
+                label={<FormattedMessage id="ui-erm-comparisons.newComparison.onDate" />}
+                name={`${name}[${index}].onDate`}
+                parser={this.parseDateOnlyString}
```

Comment:
`parseDateOnlyString` shouldnt lie within this file. Better if it lies in the `utilities` folder. Even better if its moved to `erm-components`. You can do with the former for now.

Reply:
Have done